### ISSUE TITLE
Cookies from original request cannot be combined with response cookies in session file

### ISSUE THREAD

**Issue by strindberg** [httpie/httpie#824]
Consider the following request:

```bash
https --session=/tmp/c-session "https://localhost:8721/customer/business/1" Cookie:sAuth=foo6
```

If the server sets cookies `XSRF-TOKEN` and `JSESSIONID`, the session file will look like this:

```json
{
    "__meta__": {
        "about": "HTTPie session file",
        "help": "https://httpie.org/doc#sessions",
        "httpie": "1.0.3"
    },
    "auth": {
        "password": null,
        "type": null,
        "username": null
    },
    "cookies": {
        "JSESSIONID": {
            "expires": null,
            "path": "/",
            "secure": true,
            "value": "091642DF767443D96E72C6FDEE561428"
        },
        "XSRF-TOKEN": {
            "expires": null,
            "path": "/",
            "secure": true,
            "value": "af6eb371-ce07-4583-bdce-efbfa09728f9"
        }
    },
    "headers": {
        "Cookie": "sAuth=foo6"
    }
}
```

When the request is repeated with the same session file (but without the sAuth given on the command line), the result is that only the cookie sAuth is sent, not the cookies `JSESSIONID` and `XSRF-TOKEN`:

```bash
https --verbose --session=/tmp/c-session "https://localhost:8721/apis/customer/business/1"
```

Request:

```http
GET /apis/customer/business/1 HTTP/1.1
Accept: */*
Accept-Encoding: gzip, deflate
Connection: keep-alive
Cookie: saamAuth=foo6
Host: localhost:8721
User-Agent: HTTPie/1.0.3
```

I would have expected to have all three cookies set in the request.

**Comment by kbanc** [comment]
@jakubroztocil Can I look into this?

**Comment by jakubroztocil** [comment]
@kbanc sure thing! (As always, I’d start by writing a test case reproducing the behavior and clarifying the problem.)

**Comment by isidentical** [comment]
If I am not missing anything, this seem to be fixed by @kbanc on #932. If the issue is still persisting, please let us know by creating another ticket.
